extensions/advice-slip/src/get-advice.tsx

```diff
@@ -0,0 +1,43 @@
+import { Action, ActionPanel, Detail } from "@raycast/api";
+import { useFetch } from "@raycast/utils";
+
+type Advice = {
+  slip: {
+    id: number;
+    advice: string;
+  };
+};
+
+export default function Command() {
+  const { isLoading, data, revalidate } = useFetch<Advice>("https://api.adviceslip.com/advice", {
+    keepPreviousData: false,
+    headers: {
+      Accept: "application/json",
+    },
+    async parseResponse(response) {
+      if (!response.ok) {
+        throw new Error(response.statusText);
+      }
+
+      const data = await response.text();
+      if (data !== undefined) {
+        return JSON.parse(data);
+      }
+
+      return undefined;
+    },
+  });
+
+  const joke = !isLoading && data?.slip?.advice ? data?.slip?.advice : "Loading...";
+  return (
+    <Detail
+      isLoading={isLoading}
+      markdown={`# ${joke.replaceAll(/\n/g, "\n# ")}`}
+      actions={
+        <ActionPanel>
+          <Action title="New Joke" onAction={revalidate} />
```

Comment:
Nit How about instead of "Joke", we stick with the "Advice Slip" term?
```suggestion
          <Action title="New Advice Slip" onAction={revalidate} />
```